### PR TITLE
[PRED-1395] Improve logging

### PR DESCRIPTION
**Pull request by romankolpak**
Ticket: https://datarobot.atlassian.net/browse/PRED-1395

Super small change to improve logging:
1) log version of the package
2) use better language for retries and log retry attempt count

Also, I noticed we don't ignore files such as the script output, logs, and shelves, so I've added those to `.gitignore`

@datarobot/predictions

### CODE REVIEW

File: datarobot_batch_scoring/network.py [hunk 1]
```diff
@@ -169,8 +169,8 @@ def _response_callback(self, r, batch=None, *args, **kw):
                 self.ui.error(msg)
                 self.send_error_to_ctx(batch, msg)
             else:
-                self.ui.warning('respooling batch {}'
-                                .format(batch.id))
+                self.ui.warning('retrying failed batch {}, attempts left: {}'
```
Comment (ikalnytskyi) [inline datarobot_batch_scoring/network.py]: It's more recommended way to use logging formatting style, i.e:

```
self.ui.warning('retrying failed batch %s, attempts left: %d', batch.id, batch.rty_cnt - 1)
```

The idea is to do not initiate string formatting if logging is configured to print only certain log levels. Though, in this case it won't be a performance bottleneck and I'm ok with it. Just FYI. :)

Comment (romankolpak) [inline datarobot_batch_scoring/network.py]: fun fact: batch scoring's `UI` class doesn't support this: https://github.com/datarobot/batch-scoring/blob/87148321b07c16d9b34d6818f5c89276415d75eb/datarobot_batch_scoring/utils.py#L126-L126,
it accepts just a string and no additional args. 

but you're absolutely right.

**Comment by dsakagi** [comment]
Add an `Enhancements` section to the changelog and include this - then I think you're in good shape to ship.

File: datarobot_batch_scoring/network.py (hunk 1 above)
Comment (ikalnytskyi) [inline datarobot_batch_scoring/network.py]: I thought it's logger instance, sorry. )

**Pull request merged**